docs/authzed/enterprise.md

```diff
@@ -0,0 +1,50 @@
+# Authzed Enterprise
+
+Authzed Enterprise is the on-premise installation of the exact same software powering Authzed.com.
+
+Pricing is negotiated on a per-customer basis.
```

Comment:
Pricing is a flat fee per cluster (no vCPU limits!) and is available upon request.